src/js/hooks/inboundV2/addItems/useInboundAddItemsForm.jsx

```diff
@@ -496,21 +514,22 @@ const useInboundAddItemsForm = ({
       }));
     const newSortOrder = (lineItemsData.length > 0
       ? lineItemsData[lineItemsData.length - 1].sortOrder : 0) + 100;
+
     setValue('sortOrder', newSortOrder);
     setValue('currentLineItems', getValues().isPaginated ? _.uniqBy(_.concat(data, ...getValues('currentLineItems')), 'id') : data);
-    setValue('values.lineItems', getValues().isPaginated ? _.uniqBy(_.concat(lineItemsData, ...getValues('values.lineItems')), 'id') : lineItemsData);
+    setValue('values.lineItems', getValues().isPaginated && !showOnlyImportedItems ? _.uniqBy(_.concat(lineItemsData, ...getValues('values.lineItems')), 'id') : lineItemsData);
```

Comment:
Can't the `getValues('values.lineItems')` return null? If yes, then it will throw an exception